02.recognize_digits/README.md

```diff
@@ -146,7 +146,7 @@ Here are the quick overview on the major fluid API complements.
 This is where you specify the network flow.
 1. `train_program`: A function that specify how to get avg_cost from `inference_program` and labels.
 This is where you specify the loss calculations.
-1. `optimizer`: Configure how to minimize the loss. Paddle supports most major optimization methods.
+1. `optimizer_func`: Configure how to minimize the loss. Paddle supports most major optimization methods.
```

Comment:
Can we elaborate a bit? I like the description of the `train_program`. Can we write something like "A function that specifies the configuration of the the optimizer. The optimizer is responsible for minimizing the loss and driving the training. Paddle supports many different optimizers."